scripts/ProcessTemplates.ps1

```diff
@@ -160,94 +201,234 @@ function Copy-Object {
     return $formatter
 }
 
-# Root
-#  |
-#  |- Workbooks (reportFolder)
-#        |- Performance (categoryFolder)
-#             |- Apdex (templateFolder)
-#                 |- en
-#                 |   |- settings.json
-#                 |   |- readme.md
-#                 |   |- Apdex.workbook
-#                 |- ko
-#                     |- settings.json
-#                     |- readme.md
-#                     |- Apdex.workbook        
-# -------------------------
+#----------------------------------------------------------------------------
+# CloneLocalizedRepos
+#----------------------------------------------------------------------------
+Function CloneLocalizedRepos {
+    $localSrc = Convert-Path "..\.."
+
+    foreach ($repo in $docRepos) {
+        $currentRepo = $repo.repo
+        if (Test-Path $localSrc$currentRepo) {
+            Write-Host "Trying to clone repo $currentRepo but it already exist on disk, skipping"

```

Comment:
if the repo already exists on the disk wouldn't you want to pull to get the up to date version?
or is the code guaranteed not to be on the machine when the build starts and this can get called multiple times?

Reply:
doing git pull if exist